### DISCUSSION TITLE
Adding further code notes languages

### DISCUSSION THREAD

**Discussion by good-praxis** [TriliumNext/Trilium#1392] [Q&A] [answered]
Hey folks, I am 99.9% sure that there is no way to open a PR for a new language to support through code notes on trilium (although adding the file type to the code note dropdown will require a PR), so I just wanted to ask ahead of loosing myself in the list of dependencies:

Which repos do I have to contribute to, to enable support for [GDscript](https://docs.godotengine.org/en/stable/getting_started/scripting/gdscript/gdscript_basics.html) notes in Trilium? 

I think Trilium uses [mime-types](https://github.com/jshttp/mime-types) for data file types, which in turn uses [mime-db](https://github.com/jshttp/mime-db), but which dependency takes care of the syntax highlighting?

**Comment by zadam** [comment, marked as answer]
Trilium uses CodeMirror as the code editing/highligting component.
